ProjFS.Mac/PrjFSKext/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -423,7 +424,11 @@ void ActiveProvider_Disconnect(VirtualizationRootHandle rootIndex)
         
         root->providerUserClient = nullptr;
     }
-    RWLock_ReleaseExclusive(s_rwLock);
+    RWLock_DropExclusiveToShared(s_rwLock);
```

Comment:
Pattern note: I would vote to delete the closing brace at 426 and opening brace at 428. We're not releasing the lock, so I would not close out the code block

Reply:
OK, I’ll just try to emphasise the change in lock state by surrounding it with blank lines then.